ssn/integrated/sosa-oms.ttl

```diff
@@ -307,6 +300,12 @@ sosa:Observation ogc:implements <http://www.opengis.net/spec/om/3.0/req/obs-cpt/
     schema:rangeIncludes sosa:Observation ;
     rdfs:isDefinedBy sosa-oms: .
 
+  dqv:hasQualityMeasurement
+    ogc:implements <http://www.opengis.net/spec/om/3.0/req/obs-core/AbstractObservationCharacteristics/resultQuality-sem> ;
+    skos:definition "Information pertaining to the data quality of the result." ;
+    schema:domainIncludes osa:Observation ;
```

Comment:
sosa:Observation instead of osa:Observation

I guess we'll discuss the range afterwards

Reply:
the rest looks ok to me